### PR TITLE
Bugfixes for Timers

### PR DESCRIPTION
**Pull request by kev306**
- Leaving a room after a game finished no longer attempts to vote to pause a timeout
- Client display for timer now displays correctly for certain timezones

### CODE REVIEW

**Comment by kev306** [comment]
[3906e19](https://github.com/vck3000/ProAvalon/pull/646/commits/3906e192e62d96cfc2e95f042ac81fce2fe569cc) closes #596

File: src/gameplay/game.ts [hunk 1]
```diff
@@ -1955,23 +1955,27 @@ class Game extends Room {
...
-        message: 'The game has not started. You cannot pause the timer.',
-        classStr: 'server-text',
-      });
-      return;
+      if (!leaveTriggered) {
+        socket.emit('messageCommandReturnStr', {
+          message: 'The game has not started. You cannot pause the timer.',
+          classStr: 'server-text',
+        });
+        return;
```
Comment (vck3000) [inline src/gameplay/game.ts:1963]: Return should be one line below

File: src/gameplay/game.ts [hunk 2]
```diff
@@ -1955,23 +1955,27 @@ class Game extends Room {
...
-        message: 'The game has finished. You cannot pause the timer.',
-        classStr: 'server-text',
-      });
-      return;
+      if (!leaveTriggered) {
+        socket.emit('messageCommandReturnStr', {
+          message: 'The game has finished. You cannot pause the timer.',
+          classStr: 'server-text',
+        });
+        return;
```
Comment (vck3000) [inline src/gameplay/game.ts:1977]: Same here

File: src/gameplay/game.ts [hunk 3]
```diff
@@ -1955,23 +1955,27 @@ class Game extends Room {
...
-      socket.emit('messageCommandReturnStr', {
-        message: 'The game has not started. You cannot pause the timer.',
-        classStr: 'server-text',
-      });
-      return;
+      if (!leaveTriggered) {
+        socket.emit('messageCommandReturnStr', {
+          message: 'The game has not started. You cannot pause the timer.',
+          classStr: 'server-text',
+        });
```
Comment (vck3000) [inline src/gameplay/game.ts:1962]: Let's pull out this chunk into an anonymous function (arrow function). 

```
const sendMessage = (message: string) => {
    if (!leaveTriggered) {
        socket.emit('messageCommandReturnStr', {
            message,
            classStr: 'server-text',
        });
    }
}
```

**Review by vck3000 — APPROVED** [review]

**Comment by vck3000** [comment]
Please test after you pull out that anon function :)

**Pull request merged**